.github/workflows/integration.yaml

```diff
@@ -141,3 +136,17 @@ jobs:
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
         SLACK_TITLE: PR Build ${{ github.head_ref }} ${{ job.status }}
         SLACK_COLOR: ${{ job.status }}
+
+  checksecret:
+    name: check if DOCKERHUB_USERNAME is set in github secrets
+    runs-on: ubuntu-latest
+    outputs:
+      is_DOCKERHUB_USERNAME_set: ${{ steps.checksecret_job.outputs.is_DOCKERHUB_USERNAME_set }}
+    steps:
+      - name: Check whether unity activation requests should be done
```

Comment:
Hehehe

Reply:
omg, we should just land your pr on this, this seems like overkill